src/postprocess/ked.jl

```diff
@@ -0,0 +1,20 @@
+"""
+Computes the kinetic energy density (ked), 0.5 ∑ fn |∇ψn|^2.

```

Comment:
we usually use 1/2 rather than 0.5 in the code (small detail, but might as well keep it for consistency)